utils/containers.conf

```diff
@@ -0,0 +1,12 @@
+[containers]
```

Comment:
It seems better to put this file into `utils/dockerfile_assets`

Reply:
Move the file